kogito-codegen/src/main/java/org/kie/kogito/codegen/decision/DecisionCodegen.java

```
@@ -104,19 +104,19 @@ public static DecisionCodegen ofPath(Path... paths) throws IOException {
         return ofDecisions(resources);
     }
 
-    public static DecisionCodegen ofFiles(Path basePath, List<File> files) throws IOException {
```

If you want to remove it, for me it's fine, but my previous understanding is that this was "by design" of codegenerators? possibly @evacchi and @mariofusco can better advise here.

---

if it never throws, I guess it's ok to drop that. I don't have a strong opinion there :)

---

@evacchi @tarilabs 
I think the exception could be dropped.